site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -207,6 +207,22 @@
         dropzone.addEventListener("dragleave", draghandle, false);
         dropzone.addEventListener("drop", dropWithMultipleZips, false);
     </script>
+    <script>
+        $(function() {
+            var paths = Object.keys($.parseJSON('{{ directory_priorities|json_encode|raw }}'));
```

Comment:
It is not good practice to use `var` in JS code.  You should use `let` or `const` instead, as appropriate.  The differences are discussed in a very good stack overflow post here: https://stackoverflow.com/a/11444416/6768191

```suggestion
            const paths = Object.keys($.parseJSON('{{ directory_priorities|json_encode|raw }}'));
```